_adr/81/index.adoc

```diff
@@ -19,10 +19,243 @@ NOTE: https://datatracker.ietf.org/doc/html/rfc2119[RFC-2119] keywords are used
+The fleet manager SHALL provide an endpoint which the MUST UI use to determine the available series and sizes.
+The UI MUST NOT make assumptions about linearity of scaling, instead it uses only the series and sizes provided by the endpoint.
+
+==== Series discovery
+
+Reusing the "Example" series defined above, let's illustrate how the Fleet Manager could communicate the available kinds of cluster to a UI.
+
+.Sequence diagram of UI/Fleet manager interaction
+image::ui-fleet_manager.png[UI/Fleet manager interaction]
+
+After the UI has discovered the available cloud providers and regions it would make another GET request `/api/kafkas_mgmt/v1/series/{cloudId}/{cloudRegion}` to discover the series available in this cloud and region.
+An example response JSON is shown below.
+
+[source,json]
+.Example series available to a UI from the Fleet Manager's `GET /api/kafkas_mgmt/v1/series/{cloudId}/{cloudRegion}` endpoint
+----
+"series": [ <1>
+    { "name": "Example", <2>
+      "id": 1, <3>
+      "description": "Multi-broker, multi-AZ", <4>
+      "sizes": [ <5>
+        { 
+          "name": "small", <6>
+          "parameters": [ <7>
+            { "name": "ingress", "value": "30", "unit": "MB/s" }
+            { "name": "egress", "value": "30", "unit": "MB/s" }
+            { "name": "partitions", "value": "1000" }
+            { "name": "connections", "value": "500" }
+            { "name": "connectionRate", "value": "100", "unit": "/s" }
+            { "name": "maxMessageSize", "value": "1", "unit": "MB" }
+          ]
+        },
+        { 
+          "name": "medium",
+          "parameters": [
+            { "name": "ingress", "value": "60", "unit": "MB/s" }
+            { "name": "egress", "value": "60", "unit": "MB/s" }
+            { "name": "partitions", "value": "2000" }
+            { "name": "connections", "value": "1000" }
+            { "name": "connectionRate", "value": "100", "unit": "/s" }
+            { "name": "maxMessageSize", "value": "1", "unit": "MB" }
+          ]
+        },
+        { 
+          "name": "large",
+          "parameters": [
+            { "name": "ingress", "value": "90", "unit": "MB/s" }
+            { "name": "egress", "value": "90", "unit": "MB/s" }
+            { "name": "partitions", "value": "3000" }
+            { "name": "connections", "value": "1500" }
+            { "name": "connectionRate", "value": "100", "unit": "/s" }
+            { "name": "maxMessageSize", "value": "1", "unit": "MB" }
+          ]
+        }
+      ],
+    },
+    // other series as necessary
+]
+----
+<1> This list is not considered ordered
+<2> Names MUST be unique within the `series` list 
+<3> The `id` uniquely identifies a series, allowing the name to be changed
+<4> Can be used in a UI/CLI to help guide users to appropriate profile/flavour of the service
+<5> The list MUST be ordered such that later items represent clusters with more capacity
+<6> The name MUST be unique within the `sizes` list
```

Comment:
A description field might be useful here, What do you think? @jgiardino

Reply:
Are you thinking that a description could be something we display in the UI, or used for something else? I like the idea of having descriptions for these that we could present in the UI, and having them come from the API.  But I also realize that providing descriptions that are good requires extra work.